appyters/PrismEXP/README.md

```diff
@@ -1,6 +1,9 @@
 # PrismEXP
+## Gene Function Prediction
 
-![PrismEXP](static/featured-image.png?raw=true "PrismEXP")
+<p align="center">
+<img src="static/featured-image.png" width=800 />
```

Comment:
Please use `class="img-fluid"` and don't explicitly set the width, or make it `100%`, this is relevant for how it will get displayed in the catalog landing page